packages/core/docs/performance/good-practices.md

```diff
@@ -0,0 +1,338 @@
+# Web Performance good practices
+
+Bellow you will find list of good practices that will help you optimize your website performance. 
+
+
+⚫⚫⚪ - this indicator shows how impactful each problem can be
+
+
+## Improving Core Web Vitals
+
+### Have a fast Largest Contentful Paint ⚫⚫⚫
+
+The Largest Contentful Paint (LCP) is one of the Google Web Vitals representing a time needed to display the biggest element visible to the User within the initial viewport.
+
+::: tip HINTS
+- You should preload the largest contentful paint, this will make it load faster.
+
+```javascript
+<link rel="preload" href="/path/to/image.jpg" as="image">
+```
+
+or
+
+```javascript
+head() {
+ return {
+    link: [
+      {
+        rel: 'preload',
+        as: 'image',
+        href: 'path/to/lcp/image',
+      },
+    ],
+  }
+}
+```
+
+- You should avoid lazy load images that are considered as LCP using the lazy attribute.
+
+- You can also add importance="high" to the image to increase the load priority for now work's only in Chrome (Priority Hints)
+:::
+
+### Set height and width to the images to prevent layout shift ⚫⚫⚫
+
+All images should have it's width and height set, browser when parsing html is reserving space for images, if it's dimensions are not set browser will reserve 0x0 px then when the image is loaded browser will know its size and refresh layout doing Layout Shift.
+
+## Optimizing HTML andCSS
+
+### Minify/Remove unnecessary CSS ⚫⚫⚪
+When you are building a big application, you will get to a place where your project may have much more code that it actually needs and uses.
+
+::: tip HINTS
+- Use tools like CSS Minification.
+- To eliminate unused css use a tool like PurgeCSS, or CSS Minification.
+- Vue Storefront uses PurgeCSS out of the box, you just need to setup it in config:
```

Comment:
This is wrong as we use purgeCSS in VSF different way. Please check @vue-storefront/core to see how performance module works. I thought that you wanted to put here general notes for perf issues in web dev, not in VSF. So for that, please check that and adjust the nuxt.config.js accordingly